app/views/signature/edit.html.erb

```diff
@@ -20,6 +20,14 @@
           <%= f.cfa_input_field(:signature, "") %>
         </div>
       </div>
+      <%= f.cfa_radio_set_with_follow_up(:did_you_get_help,
+                          label_text: t("signature.did_you_get_help"),
```

Comment:
Picky as all get out, but Rubocop didn't catch the double quotes here

Reply:
cause rubocop doesn't look at erb files....lol